exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter.h

```diff
@@ -19,14 +21,16 @@ const size_t MAX_BUFFER_SIZE = 100;
  * A in memory exporter that switches a flag once a valid recordable was received
  * and keeps track of all received spans in memory.
  */
+using InMemorySpanData = exporter::memory::InMemoryData<sdk::trace::SpanData>;
```

Comment:
should we instead have `InMemorySpanData` as a class to inherit from `InMemoryData`? And also have `InMemorySpanData::GetSpans()` method. This way applications using span memory exporter won't need to change.

Reply:
Agree

Reply:
Thanks! Updated the PR to inherit from the InMemoryData. PTAL.